docs/requirements.txt

```diff
@@ -1,5 +1,2 @@
-mkdocs<1.1
-jinja2==3.0.0
-mkdocs-awesome-pages-plugin==2.1.0
-markdown<3.4.0
-readthedocs-version-warning-mkdocs-plugin
+mkdocs==1.6.1
+mkdocs-awesome-pages-plugin==2.10.1
```

Comment:
We should implement some package manager like poetry instead of using `requirements.txt`

Reply:
I don't find it quite necessary here